content/lessons/2016/inscribed-rectangle-problem/index.md

```diff
@@ -1,10 +1,320 @@
+
+{{< figure image="torus-and-loop-correspondence.png" >}}
+There is a unique correspondence between pairs of points on the loop and individual points on the torus.
+{{< /figure >}}
+
+The torus is to pairs of points on the loop as the xy-plane is to pairs of points on the real number line.
+
+{{< figure image="coordinate-plane-analogy.png" >}}
+The torus serves as a 2D surface for encoding pairs of positions on a loop, much like how the xy-plane encodes pairs of positions on the number line.
+{{< /figure >}}
+
+The key property of this association is that it’s continuous both ways, meaning if you nudge any point on the torus by just a tiny amount, it corresponds to only a slight nudge in the pair of points on the loop, and vice versa.
+
+{{< question
+  question="What does a single point on the torus correspond to?"
+  choice1="A unique point on the loop"
+  choice2="A unique ordered pair of points on the loop"
+  choice3="A unique unordered pair of points on the loop"
+  choice4="A unique pair of points in the xy-plane"
+  correct=2
+>}}
+
+Now we’re really getting into the heart of what topology is all about. By representing a set of things we care about, like pairs of points on a loop, as individual points on a surface, we can begin tackling our original question by understanding various properties of this surface. But the torus is not *quite* the surface that you and I care about today.
+
+{{< section >}}
+### Unordered Pairs As a Surface
+If the torus is the natural shape for *ordered* pairs of points on the loop, what’s the natural shape for *unordered* pairs?
+
+Remember, the whole reason we’re doing this is to show that two distinct pairs of points on the loop share a midpoint and are the same distance apart.  But if we consider the pair $(a, b)$ to be distinct from $(b, a)$, that would give us two separate pairs which trivially have the same midpoint and distance apart! That’s like saying you can always find a rectangle as long as you consider any pair of points to be a rectangle. Not helpful!
+
+To find a surface representing *unordered* pairs, let’s look back at our unit square. We want the coordinates $(0.2, 0.3)$, say, to represent the same pair as $(0.3, 0.2)$. In general the coordinates $(x, y)$ should represent the same thing as $(y, x)$.
+
+{{< figure image="matching-points.png" >}}
+Since we’re interested in unordered pairs, the points $(x, y)$ and $(y, x)$ on the square both represent the same pair of points on the loop.
+{{< /figure >}}
+
+Once again, we capture this idea by gluing points together when they’re supposed to represent the same pair, which in this case requires folding the square diagonally.
+
+{{< figure image="fold-square.png" video="fold-square.mov" >}}
```

Comment:
Again, for some reason, this clip does not show for me.  I wonder if it has anything to do with using .mov instead of .mp4?  It shouldn't...